internal/internal_workflow.go

```diff
@@ -784,6 +801,10 @@ func (c *channelImpl) ReceiveAsyncWithMoreFlag(valuePtr interface{}) (ok bool, m
 	}
 }
 
+func (c *channelImpl) Empty() bool {
```

Comment:
I think a `Len() int` and a `Cap() int` here is more in line with https://pkg.go.dev/builtin#len and https://pkg.go.dev/builtin#cap respectively.